Automattic/wp-calypso
# A4A: Update Pressable logos.

Pressable has launched its new logo, and this PR updates any Pressable logo reference in A4A to use the new one.

Closes https://github.com/Automattic/automattic-for-agencies-dev/issues/844

## Proposed Changes

* Replace the Pressable logo images to use the new one. Fix a couple of broken UI after replacing the newer and bigger images.

## Why are these changes being made?


* We need to match the Pressable logo to have consistent branding.

## Testing Instructions

* Use the A4A live link below and test all A4A pages that have the Pressable logo on them and confirm it isn't broken.

| Area | Before | After |
|--------|--------|--------|
| `/overview` | <img width="210" alt="Screenshot 2024-07-25 at 12 07 42 AM" src="https://github.com/user-attachments/assets/af78becc-bf9a-4f34-968f-1cb301c672e3"> | <img width="243" alt="Screenshot 2024-07-25 at 12 07 49 AM" src="https://github.com/user-attachments/assets/3802a4ce-9b2d-4789-94bb-bf73010a32f2"> |
| `/marketplace/hosting?flags=-a4a-hosting-page-redesign` | <img width="358" alt="Screenshot 2024-07-25 at 12 10 38 AM" src="https://github.com/user-attachments/assets/0aa46d97-1b2a-4c11-9f41-e2fdfcd3af94"> | <img width="400" alt="Screenshot 2024-07-25 at 12 10 05 AM" src="https://github.com/user-attachments/assets/2a3dd07e-d0bc-4257-8038-e5cd46e126b0"> |
| `/sites/overview/{PRESSABLE_SITE}/hosting-overview` | <img width="346" alt="Screenshot 2024-07-25 at 12 17 47 AM" src="https://github.com/user-attachments/assets/0be14342-d90b-4c5c-bc3d-12b9aeb2b979"> | <img width="322" alt="Screenshot 2024-07-25 at 12 20 53 AM" src="https://github.com/user-attachments/assets/4efb6788-c2b6-42a0-ae73-44b1aba31c2b"> | 
| `/marketplace/hosting/pressable` | <img width="689" alt="Screenshot 2024-07-25 at 12 11 39 AM" src="https://github.com/user-attachments/assets/b4c12b5e-6510-443e-9b3d-bc9c6219b6eb"> | <img width="583" alt="Screenshot 2024-07-25 at 12 11 45 AM" src="https://github.com/user-attachments/assets/384f4977-b35f-4d51-9400-8956be059b89"> | 
| `/migrations` | <img width="376" alt="Screenshot 2024-07-25 at 12 13 16 AM" src="https://github.com/user-attachments/assets/0d6756be-b540-4e62-9b46-7e7a3f5a48b0"> | <img width="388" alt="Screenshot 2024-07-25 at 12 13 03 AM" src="https://github.com/user-attachments/assets/d9541e4b-7685-4bdf-98d6-9bf4aeb77ec1"> | 
| `/partner-directory/dashboard` | <img width="725" alt="Screenshot 2024-07-25 at 12 16 20 AM" src="https://github.com/user-attachments/assets/74267f78-c941-40ff-a533-f4b54918b25e"> | <img width="722" alt="Screenshot 2024-07-25 at 12 15 05 AM" src="https://github.com/user-attachments/assets/8aed2987-4981-48d0-97fb-456998d0c99c"> | 
| `/referrals/faq` | <img width="211" alt="Screenshot 2024-07-25 at 12 20 07 AM" src="https://github.com/user-attachments/assets/75991323-ace4-43b6-a884-da1de508eabb"> | <img width="206" alt="Screenshot 2024-07-25 at 12 19 52 AM" src="https://github.com/user-attachments/assets/8dad82e8-b9d0-4766-959a-21a6e15aa2cc"> | 
| `/referrals/dashboard` | <img width="157" alt="Screenshot 2024-07-25 at 12 23 16 AM" src="https://github.com/user-attachments/assets/e0494e16-6b75-4fcb-8c3c-936a7b2da3d3"> | <img width="192" alt="Screenshot 2024-07-25 at 12 22 27 AM" src="https://github.com/user-attachments/assets/00a92dc6-cb23-41c7-83e0-5e7f6017bc3c"> | 


## Pre-merge Checklist
